diff --git a/src/c47/keyboard.c b/src/c47/keyboard.c
index bc44026016e94a8865b8acc886386341596641c6..6cd01d7bccc5f6c3e6c3cdc4d59eef6dd6635fa9 100644
--- a/src/c47/keyboard.c
+++ b/src/c47/keyboard.c
@@ -3577,6 +3577,11 @@ void fnKeyEnter(uint16_t unusedButMandatoryParameter) {
         break;
       }
 
+      case CM_PRETTY_BROWSER: {
+        prettyBrowserEnter();
+        break;
+      }
+
       case CM_TIMER: {
         fnRegAddTimerApp(NOPARAM);     //ENTER
         break;
@@ -3938,6 +3943,11 @@ void fnKeyExit(uint16_t unusedButMandatoryParameter) {
         break;
       }
 
+      case CM_PRETTY_BROWSER: {
+        prettyBrowserLeave();
+        break;
+      }
+
       case CM_ASN_BROWSER: {
         if(previousCalcMode == CM_AIM || tam.alpha) {
           if(currentMenu() == -MNU_AIMCATALOG) {
@@ -4348,6 +4358,11 @@ void fnKeyBackspace(uint16_t unusedButMandatoryParameter) {
         break;
       }
 
+      case CM_PRETTY_BROWSER: {
+        prettyBrowserLeave();
+        break;
+      }
+
       case CM_BUG_ON_SCREEN:
       case CM_LISTXY:
       case CM_GRAPH:
@@ -4612,6 +4627,11 @@ void fnKeyUp(uint16_t unusedButMandatoryParameter) {
         break;
       }
 
+      case CM_PRETTY_BROWSER: {
+        prettyBrowserUp();
+        break;
+      }
+
       case CM_PEM: {
         resetAlphaSelectionBuffer();
         if(getSystemFlag(FLAG_ALPHA) && alphaCase == AC_LOWER) {
@@ -4830,6 +4850,11 @@ void fnKeyDown(uint16_t unusedButMandatoryParameter) {
         break;
       }
 
+      case CM_PRETTY_BROWSER: {
+        prettyBrowserDown();
+        break;
+      }
+
       case CM_PEM: {
         resetAlphaSelectionBuffer();
         if(getSystemFlag(FLAG_ALPHA) && alphaCase == AC_UPPER) {
@@ -4931,6 +4956,11 @@ void fnKeyDotD(uint16_t unusedButMandatoryParameter) {
         break;
       }
 
+      case CM_PRETTY_BROWSER: {
+        prettyBrowserPan();
+        break;
+      }
+
       case CM_REGISTER_BROWSER:
       case CM_FLAG_BROWSER:
       case CM_ASN_BROWSER:
