diff --git a/src/c47/items.h b/src/c47/items.h
index 98924313217f3b57f1b663367c8e45c0dc0f1010..453aa53ee5ae2dd485ed8f5098031bd9eb08ed46 100644
--- a/src/c47/items.h
+++ b/src/c47/items.h
@@ -446,9 +446,9 @@
 #define ITM_SDR                      424
 #define ITM_ZIP                      425
 #define ITM_UNZIP                    426
-#define ITM_0427                     427
-#define ITM_0428                     428
-#define ITM_0429                     429
+#define ITM_UHIST                    427
+#define ITM_REDO                     428
+#define ITM_HCLR                     429
 #define ITM_0430                     430
 #define ITM_0431                     431
 #define ITM_0432                     432
@@ -2351,7 +2351,7 @@
 #define SFL_PDIFF                   2296
 #define SFL_PSHADE                  2297
 #define SFL_SBadm                   2298
-#define SFL_2299                    2299
+#define SFL_UHIST                   2299
 #define SFL_2300                    2300
 #define SFL_2301                    2301
 #define SFL_2302                    2302
