diff --git a/src/c47/c47Extensions/keyboardTweak.c b/src/c47/c47Extensions/keyboardTweak.c
index c1f0225c46399f4e0f6e3c92e4d43989937a88ed..b7b4c627a4ce2937e1e4cfa749a2bc7a2dd03e66 100644
--- a/src/c47/c47Extensions/keyboardTweak.c
+++ b/src/c47/c47Extensions/keyboardTweak.c
@@ -947,7 +947,7 @@ void resetKeytimers(void) {
       screenUpdatingMode = screenUpdatingModeMeM;      //clear skip that was set in resetShiftState()
       screenUpdatingMode &= ~SCRUPD_MANUAL_STATUSBAR;
 
-      if(!(calcMode == CM_REGISTER_BROWSER || calcMode == CM_FLAG_BROWSER || calcMode == CM_ASN_BROWSER || calcMode == CM_FONT_BROWSER || GRAPHMODE || calcMode == CM_LISTXY)) {
+      if(!(calcMode == CM_REGISTER_BROWSER || calcMode == CM_FLAG_BROWSER || calcMode == CM_ASN_BROWSER || calcMode == CM_FONT_BROWSER || calcMode == CM_HIST_BROWSER || GRAPHMODE || calcMode == CM_LISTXY)) {
         if((calcMode == CM_ASSIGN && itemToBeAssigned == 0) || FN_timed_out_to_NOP_or_Executed) { //Clear any possible underline residues
           showSoftmenuCurrentPart();
         }
