diff --git a/src/c47/defines.h b/src/c47/defines.h
index ed231842bb47fe69aa7babe93bc67243dab21e00..ab85a537c21c9f0a69617b56b56f397e12d94187 100644
--- a/src/c47/defines.h
+++ b/src/c47/defines.h
@@ -1435,7 +1435,7 @@ static inline uint8_t regCtoKS(const int16_t regC) {
 #define Y_POSITION_OF_REGISTER_Y_LINE             96
 #define Y_POSITION_OF_REGISTER_X_LINE            132
 
-#define NUMBER_OF_DYNAMIC_SOFTMENUS               22
+#define NUMBER_OF_DYNAMIC_SOFTMENUS               23  // forth-core P-H6
 #define SOFTMENU_HEIGHT                           23
 
 
