diff --git a/src/c47/programming/nextStep.c b/src/c47/programming/nextStep.c
index 22c0014a066ae8f215c052f7fd80133eb6bb4692..db5e925a3e20d6d3d6a200a94fb1ee650a41a870 100644
--- a/src/c47/programming/nextStep.c
+++ b/src/c47/programming/nextStep.c
@@ -7,6 +7,7 @@
  ***********************************************/
 
 #include "c47.h"
+#include "forth_capture.h"
 
 
 bool_t programBytesAvailable(const uint8_t *address, uint16_t numberOfBytes) {
@@ -451,7 +452,7 @@ static void _bstInPem(void) {
 void fnBst(uint16_t unusedButMandatoryParameter) {
   screenUpdatingMode = SCRUPD_AUTO;
   if(calcMode == CM_PEM) {
-    if(aimBuffer[0] != 0) {
+    if(aimBuffer[0] != 0 || forthCapTextNonEmpty()) {
       if(getSystemFlag(FLAG_ALPHA)) {
         pemCloseAlphaInput();
       }
@@ -524,7 +525,7 @@ void showStep(void) {
 void fnSst(uint16_t unusedButMandatoryParameter) {
   screenUpdatingMode = SCRUPD_AUTO;
   if(calcMode == CM_PEM) {
-    if(aimBuffer[0] != 0) {
+    if(aimBuffer[0] != 0 || forthCapTextNonEmpty()) {
       if(getSystemFlag(FLAG_ALPHA)) {
         pemCloseAlphaInput();
       }
