diff --git a/src/c47/c47Extensions/addons.c b/src/c47/c47Extensions/addons.c
index 26d98873518bcec7a4eb2bc5a9bd2f069318300c..7092b0adf66b0d94711bfa5aa9925a2502d8aa68 100644
--- a/src/c47/c47Extensions/addons.c
+++ b/src/c47/c47Extensions/addons.c
@@ -663,7 +663,7 @@ void fnEdit (uint16_t unusedParamButMandatory) {
         //printf("**[DL]** fnEdit cmPem func %d opParam %d opParam2 %d decodedLiteralType %d\n", func, opParam, opParam2, decodedLiteralType);
         //fflush(stdout);
 
-        if((func == ITM_LITERAL) || (func == ITM_REM)) {
+        if((func == ITM_LITERAL) || (func == ITM_REM) || (func == ITM_FORTH)) {
           memset(aimBuffer, 0, AIM_BUFFER_LENGTH);
 
           if(opParam == STRING_LABEL_VARIABLE) {
