diff --git a/src/c47/printing/print.c b/src/c47/printing/print.c
index 2e525c664ac875fdcae8a637d79824fe4080dc26..634a09c8002822bb0eecdf8f457253fcf8f2355c 100644
--- a/src/c47/printing/print.c
+++ b/src/c47/printing/print.c
@@ -1229,7 +1229,6 @@ uint8_t reverse(uint8_t b) {
 //
 void printLcd(void) {
   int32_t x, y;
-  uint8_t * line_addr = 0;
   uint8_t data;
   int16_t i;
   int32_t offset;
@@ -1251,11 +1250,7 @@ void printLcd(void) {
     sendByteIR((x == 0 ? 0xf1 : 0xff));
     // print screen line
     for(y= SCREEN_HEIGHT-1; y>=0; y--) {
-      #if defined(DMCP_BUILD)
-        line_addr = lcd_line_addr(y);
-      #elif defined(PC_BUILD) // !DMCP_BUILD
-        // To be coded
-      #endif // DMCP_BUILD
+      uint8_t *line_addr = lcd_line_addr(y);
 
       for(i=2; i>= 0; i--) {
         offset = 3*x + i;
@@ -2580,4 +2575,3 @@ void fnP_PrintAllItems (uint16_t unusedButMandatoryParameter) {
     }
   #endif //PC_BUILD && OPTION_IR_PRINTING
 }
-
