diff --git a/src/testSuite/hal/lcd.c b/src/testSuite/hal/lcd.c
index 9148d44dea49c73b03c87d31aef8d1a96cf7a5b6..3252b30449761cbd8ee7544a7ff590d93e9ad812 100644
--- a/src/testSuite/hal/lcd.c
+++ b/src/testSuite/hal/lcd.c
@@ -19,6 +19,18 @@ static void ensureLcdBuffer(void) {
   }
 }
 
+uint8_t *lcd_line_addr(int row) {
+  ensureLcdBuffer();
+  if(row < 0 || row >= SCREEN_HEIGHT) {
+    char tmp[1000];
+    sprintf(tmp, "row = %" PRIi32 ", it should be >= 0 and < %" PRIi32 "!\n", (int32_t)row, (int32_t)SCREEN_HEIGHT);
+    abortf(tmp);
+  }
+
+  lcd_buffer[52 * row] = 1u;
+  return lcd_buffer + 52 * row + 2;
+}
+
 void bitblt24(uint32_t x, uint32_t dx, uint32_t y, uint32_t val, int blt_op, int fill) {
   ensureLcdBuffer();
   if(dx < 1 || dx > 24) {
