diff --git a/src/c47/error.c b/src/c47/error.c
index 382c32553be6a8cdfe1d954145262ff68673ab8c..a9eb983476b511356aa4941c40c96c6778d03039 100644
--- a/src/c47/error.c
+++ b/src/c47/error.c
@@ -285,6 +285,9 @@ void displayCalcErrorMessage(uint8_t errorCode, calcRegister_t errMessageRegiste
        if(lastErrorCode == ERROR_RESERVED_VARIABLE_NAME) {
          sprintf(tmpString, "%s: %s", errorMessages[lastErrorCode], errorMessage);
        }
+       else if(lastErrorCode == ERROR_FUNCTION_NOT_FOUND && errorMessage[0]) {
+         sprintf(tmpString, "%s: %s", errorMessages[lastErrorCode], errorMessage);
+       }
        else {
          sprintf(tmpString, "%s", errorMessages[lastErrorCode]);
        }
