diff --git a/src/c47/c47Extensions/addons.c b/src/c47/c47Extensions/addons.c
index 26d98873518bcec7a4eb2bc5a9bd2f069318300c..ba1e4e2fbef2020173d069d71d0d6f58f530effb 100644
--- a/src/c47/c47Extensions/addons.c
+++ b/src/c47/c47Extensions/addons.c
@@ -1367,7 +1367,10 @@ void fnFrom_ms(uint16_t unusedButMandatoryParameter){
 
 
 void fnTo_ms(uint16_t unusedButMandatoryParameter) {
-    switch(calcMode) { //JM
+    switch(pgEffectiveCalcMode()) { //JM   // program-graphics: see fnKeyEnter.
+      case CM_GRAPHICS_CANVAS: {   // program-graphics: ignore keyboard in canvas view.
+        break;
+      }
       case CM_NIM:
         addItemToNimBuffer(ITM_ms);
         break;
